pyproject.toml

```diff
@@ -6,9 +6,10 @@ requires = [
     "wheel",
     "Cython>=0.29.21,<3",  # Note: sync with setup.py
     "numpy==1.16.5; python_version=='3.7' and platform_system!='AIX'",
-    "numpy==1.17.3; python_version>='3.8' and platform_system!='AIX'",
+    "numpy==1.17.3; python_version=='3.8' and platform_system!='AIX'",
     "numpy==1.16.5; python_version=='3.7' and platform_system=='AIX'",
-    "numpy==1.17.3; python_version>='3.8' and platform_system=='AIX'",
+    "numpy==1.17.3; python_version=='3.8' and platform_system=='AIX'",
```

Comment:
doesn't this duplicate the line?

Reply:
I don't see it. Please elaborate...

Reply:
are L9 and L11 the same? is there a reason

Reply:
It's `==` vs. `!=`.